frontend/src/components/Input/AddAttachmentInput.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useRef } from 'react';
+import { MdAdd } from 'react-icons/md';
+
+const AddAttachmentsInput = ({ onFileUpload }) => {
+  const [files, setFiles] = useState([]); // Store multiple files
+  const fileInputRef = useRef(null); // Create a ref for the file input
+
+  const handleFileChange = (event) => {
+    const selectedFiles = Array.from(event.target.files); // Get the selected files
+    setFiles(selectedFiles); // Update the state with the selected files
+  };
+
+  const handleButtonClick = () => {
+    if (files.length > 0) {
+      console.log('Files selected:', files);
```

Comment:
we should avoid console logs in production

Reply:
I'll take care of this next time.